pkg/testutil/golden/golden.go

```diff
@@ -0,0 +1,28 @@
+// Package golden provides function and helpers to use golden file for
+// testing purpose.
+package golden
+
+import (
+	"flag"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+)
+
+var update = flag.Bool("test.update", false, "update golden file")
```

Comment:
yeah same for me, but as it's a testing package I thought it was a trade-off I was willing to take.